backend/Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM gclub/skywardai:v0.1.2
+FROM gclub/skywardai:v0.1.3
 
 WORKDIR /app
 
 EXPOSE 8000
 
+# Install Python dependencies
+COPY ./requirements.txt ./
```

Comment:
We install all the requirements in base image. There may not need to install the dependencies again